### PR TITLE
CeloCLI query attestations

### PR DESCRIPTION
**Pull request by codyborn**
### Description

CeloCli command to query attestations for an account. Uses --from account to query ODIS pepper if not passed in.

### Other changes

Built on @bowd's branch: 
https://github.com/celo-org/celo-monorepo/pull/5703/files

### Tested

Manually tested locally.

### CODE REVIEW

File: packages/cli/src/commands/account/get-attestations.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        console.error('Must specify phoneNumber if identifier not provided')
+      }
+      // Get Phone number pepper
+      // Needs a balance to perform query
+      if (!pepper) {
+        pepper = await this.getPhoneNumberPepper(this.kit, phoneNumber!, account!, network)
+        console.log('Pepper: ' + pepper)
+      }
+
+      const computedIdentifier = web3.utils.soliditySha3({
```
Comment (medhak1) [inline packages/cli/src/commands/account/get-attestations.ts:68]: nit: would suggest using the kit for this, just to standardize how we call web3 functions
Then, you can also remove the web3 variable on line 54. 

```suggestion
      const computedIdentifier = this.kit.connection.web3.utils.soliditySha3({
```

Comment (medhakothari) [inline packages/cli/src/commands/account/get-attestations.ts]: nit: would suggest using the kit for this, just to standardize how we call web3 functions
Then, you can also remove the web3 variable on line 54. 

```suggestion
      const computedIdentifier = this.kit.connection.web3.utils.soliditySha3({
```

File: packages/cli/src/commands/account/get-attestations.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    const account = res.flags.from
+    let identifier = res.flags.identifier
+    let pepper = res.flags.pepper
+    if (!account && !pepper && !identifier) {
+      console.error('Must specify either --from or --pepper or --identifier')
+      return
+    }
+    const network = res.flags.network
+
+    const web3 = new Web3()
```
Comment (medhakothari) [inline packages/cli/src/commands/account/get-attestations.ts]: see above comment

```suggestion
```

File: packages/cli/src/commands/account/get-attestations.ts [hunk 3]
```diff
@@ -0,0 +1,99 @@
+import { ContractKit } from '@celo/contractkit'
+import { OdisUtils } from '@celo/identity'
+import { AuthSigner } from '@celo/identity/lib/odis/query'
+import { flags } from '@oclif/command'
+import Web3 from 'web3'
```
Comment (medhakothari) [inline packages/cli/src/commands/account/get-attestations.ts]: see above comment
```suggestion
```

**Review by medhakothari — CHANGES_REQUESTED** [review]
Everything looks good to go! Just added 1 small nit, and also was wondering if you can add a test for this command. Trying to increase test coverage in the CLI one command at a time :)

**Review by medhakothari — APPROVED** [review]
LGTM! 🚀

**Pull request merged**